src/components/AboutMe/styles/index.css

```diff
@@ -21,6 +21,41 @@
 	animation-fill-mode: forwards;
 }
 
+.sam-profile-pic {
+	height: 180px;
+	margin-top: 3rem;
+	border-radius: 35px;
+}
+
+.description-sam {
```

Comment:
NIT: Lets try to be consistent with your naming either stick to post pending sam or pre pending dont mix both like this line and line 36 sam-glow